tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,29 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+
+class BugzillaLite(Base):
+
+    name = 'Bzlite'
+    _given_username = ('css selector', "#login input[type='email']")
```

Comment:
We have a constant for 'css selector' in Marionette. You can use By.CSS_SELECTOR [like here](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/messages/app.py#L17)